core-web/libs/template-builder/src/lib/components/template-builder/components/add-widget/add-widget.component.ts

```diff
@@ -0,0 +1,73 @@
+import {
+    ChangeDetectionStrategy,
+    ChangeDetectorRef,
+    Component,
+    HostListener,
+    Input
+} from '@angular/core';
+
+import { colIcon, rowIcon } from './icons';
+
+type WidgetType = 'col' | 'row';
+
+const iconsMap = {
+    row: rowIcon,
+    col: colIcon
+};
```

Comment:
should we move this to to `models.ts`

Reply:
That's not a type. It's a map for the icons, which are very specific on the design, so if they are not used in other components, we can leave it at a `component level` for now.